### DISCUSSION TITLE
运行哪个文件会有huggingface上面体验的效果

### DISCUSSION THREAD

**Discussion by gaogaomie** [RapidAI/TableStructureRec#103] [Q&A]
https://huggingface.co/spaces/Joker1212/TableDetAndRec
这个体验的demo我想在本地运行下，怎么操作，我是小白，不是很懂这个，请指教！！！

**Comment by Joker1212** [comment]
那个是搭建的demo工程哈，你可以直接拉取demo工程的仓库代码运行，直接python app.py就行了
<img width="1652" alt="image" src="https://github.com/user-attachments/assets/287e5c5b-ea9c-45dc-83fc-ba0d63cd9d59" />
<img width="1658" alt="image" src="https://github.com/user-attachments/assets/f28ee3e9-2f72-4e4e-a63b-c73637aba503" />

**Reply by gaogaomie** [reply]
好的，非常感谢
